doc/api/url.md

```diff
@@ -1007,6 +1007,11 @@ The formatting process operates as follows:
 ### url.parse(urlString[, parseQueryString[, slashesDenoteHost]])
 <!-- YAML
 added: v0.1.25
+changes:
+  - version: v9.0.0
+    pr-url: https://github.com/nodejs/node/pull/13606
+    description: The returned URL object `search` property is now null when
```

Comment:
nit: object’s

Reply:
Thanks, I looked below and made this consistent with the `parseQueryString`.

> If `false`, the `query` property on the returned URL object will be an unparsed, undecoded string. Defaults to `false`.